src/internal/internalAPIs.ts

```diff
@@ -1,121 +1,7 @@
-import { navigateBack } from '../../src/public/navigation';
-import { LoadContext } from '../public/interfaces';
-import { validOriginRegExp, userOriginUrlValidationRegExp, defaultSDKVersionForCompatCheck } from './constants';
+import { userOriginUrlValidationRegExp, defaultSDKVersionForCompatCheck } from './constants';
 import { GlobalVars } from './globalVars';
-import { MessageResponse, MessageRequest, ExtendedWindow, DOMMessageEvent } from './interfaces';
 import { generateRegExpFromUrls, compareSDKVersions } from './utils';
 
-// ::::::::::::::::::::MicrosoftTeams SDK Internal :::::::::::::::::
-GlobalVars.handlers['themeChange'] = handleThemeChange;
-GlobalVars.handlers['fullScreenChange'] = handleFullScreenChange;
-GlobalVars.handlers['backButtonPress'] = handleBackButtonPress;
-GlobalVars.handlers['load'] = handleLoad;
-GlobalVars.handlers['beforeUnload'] = handleBeforeUnload;
-GlobalVars.handlers['changeSettings'] = handleChangeSettings;
-GlobalVars.handlers['startConversation'] = handleStartConversation;
-GlobalVars.handlers['closeConversation'] = handleCloseConversation;
-GlobalVars.handlers['appButtonClick'] = handleAppButtonClick;
-GlobalVars.handlers['appButtonHoverEnter'] = handleAppButtonHoverEnter;
-GlobalVars.handlers['appButtonHoverLeave'] = handleAppButtonHoverLeave;
-
```

Comment:
we can keep all handlers to its own nameSpace? may be "EventHandlers" instead of putting to communication. We should try to make communication nameSpace independent of handler so minimize future changes in that file.

Reply:
See PR #503. It takes care of handlers